pkg/storagev2/storagetest/batch.go

```diff
@@ -0,0 +1,114 @@
+// Copyright 2022 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package storagetest
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	storage "github.com/ethersphere/bee/pkg/storagev2"
+)
+
+func TestBatch(t *testing.T, s storage.Store) {
+	t.Run("duplicates are rejected", func(t *testing.T) {
+		b, _ := s.Batch(context.Background())
+
+		item1 := &obj1{
```

Comment:
could `item1` and `item2` variables be defined in common scope for all test cases, so those could be reused?